\documentclass{article}

\input{../figure-defs.tex}
\begin{document}

%%%%
%% challenge 1
%%%%

\inclcrop{challenge1_src/images/WTC11-sqanti-cats.pdf}{55pt}{Extended Data Fig. 1}{
  SQANTI3 classifications of LRGASP submissions on the WTC11 datset.
  a) Comparison of the number of known genes to transcripts in those genes for the WTC11
  dataset.
  b) Percentage of FSM (Full Splice Match) vs ISM (Incomplete Splice Match).
  c) Percentage of NIC (Novel In Catalog) vs NNC (Novel Not in Catalog).
  d) Percentage of known and novel transcripts with full support at junctions and end positions.
  Ba: Bambu, FM: FLAMES, FL: FLAIR,
  IQ: IsoQuant, IT: IsoTools, IB: Iso_IB, Ly: LyRic, Ma: Mandalorion, TL: TALON-LAPA, Sp:
  Spectra, ST: StringTie2.
}
\inclcrop{challenge1_src/generated/Extended_Fig._23_TMs-coverage-by-tool.pdf}{38pt}{Extended Data Fig. 2}{
  % was: Extended Data Fig. 23.
  Percentage of transcript models (TM) with different ranges of sequence
  coverage by long reads. a) WTC11. c) H1-mix. c) Mouse ES. Ba: Bambu, FM: FLAMES, FL: FLAIR,
  IQ: IsoQuant, IT: IsoTools, IB: Iso_IB, Ly: LyRic, Ma: Mandalorion, TL: TALON-LAPA, Sp:
  Spectra, ST: StringTie2.
}
\inclimg{challenge1_src/images/Coverage_all_SIRVs.steps.with-labels.ai.pdf}{Extended Data Fig. 3}{
  % was: Extended Data Fig. 42a. (updated version)
  Positional coverage of long unspliced SIRV transcript sequences by long reads
  for each sample type.
}
\inclcrop{challenge1_src/generated/Extended_Fig._45_gencode-manual-props-wtc11.pdf}{38pt}{Extended Data Fig. 4}{
  % was: Extended Data Fig. 45.
  Properties of
  GENCODE manually annotated loci for WTC11 sample.a) Distributon of gene expression. b)
  Distribution of SQANTI categories. c) Intersection of Unique Intron Chains (UIC) among
  experimental protocols.
}
\inclcrop{challenge1_src/generated/Extended_Fig._46_gencode-manual-props-es.pdf}{38pt}{Extended Data Fig. 5}{
  % was: Extended Data Fig. 46.
  Properties of GENCODE manually annotated loci for mouse ES sample.a)
  Distributon of gene expression. b) Distribution of SQANTI categories. c) Intersection of
  Unique Intron Chains (UIC) among experimental protocols.
}
%%%%
%% challenge 2
%%%%
\inclimg{challenge2_src/fig_rador_plot.pdf}{Extended Data Fig. 6}{
  % was: Extended Data Fig. 53.
  Radar plot of overall evaluation results of eight quantification tools with
  seven protocols-platforms on four data scenarios: real data with multiple
  replicates, cell mixing experiment, SIRV-set 4 data and simulation data. To
  display the evaluation results more effectively, we normalized all metrics
  to 0-1 range: 0 corresponds to the worst performance and 1 corresponds to
  the best performance.
}
\inclcrop{challenge2_src/fig_ranking_on_six_protocols_and_platforms.pdf}{58pt}{Extended Data Fig. 7}{
  % was: Extended Data Fig. 5300.
  Top 3 performance on quantification tools under six different
  protocols-platforms for each metric. Here, quantification tools showcase
  scores under six different protocols-platforms across various evaluation
  metrics, with the top 3 performers highlighted for each metric. Blank spaces
  denote instances where the tool or protocols-platforms did not have
  participants submitting the corresponding quantitative results.
}
%%%%
%% challenge 3
%%%%
\inclcrop{challenge3_src/Extended_Fig._65.pdf}{38pt}{Extended Data Fig. 8}{
  % was: Extended Data Fig. 65.
  SQANTI category classification of transcript models detected by the same
  tools in Challenge 1 predictions using the reference
  annotation and Challenge 3 predictions did not. Ba = Bambu, IQ =
  StringTie2/IsoQuant.
}
%%%%
%% validation
%%%%
\inclimg{validation_src/Extended_Data_Fig68_validation_by_supportive_reads.pdf}{Extended Data Fig. 9}{
  % was: Extended Data Fig. 68.
  Fraction of experimentally validated WTC-11 transcripts as a function of the total numbers
  of long reads that were observed across the 21 library preparations (e.g.,
  PacBio cDNA, ONT cDNA, PacBio CapTrap).
}
\end{document}
