\documentclass{article}

\input{../figure-defs.tex}
\begin{document}

%%%%
% Overview
%%%%
\inclimgsz{overview_src/summary-of-data.pdf}{140mm}{Supplementary Fig. 1.}{
  Summary of LRGASP Data.
}
\inclimgsm{overview_src/challenge-submission.pdf}{Supplementary Fig. 2.}{
  Challenge submission. a) Overview of submissions to Challenges 1 and 2. Each
  entry was derived from a specific data category, library prep, and
  sequencing platform combination. All available samples for the selected
  combination must be included in an entry. b) Overview of submissions for
  Challenge 3.
}
\inclimg{overview_src/submission-structure.pdf}{Supplementary Fig. 3.}{
  Schematic of directory structure and files that would be included in each entry.
}
\inclimgsm{overview_src/challenge1-flow.pdf}{Supplementary Fig. 4.}{
  Flow diagram of Challenge 1: Transcript isoform detection with a
  high-quality genome. Samples, library prep methods, and sequencing platforms
  used in the challenge are indicated at the top. Participants select which
  data category, library prep, and sequencing platform to analyze, run their
  pipelines to generate transcript predictions, and submit an entry which
  includes predictions for all samples. The entries include a GTF file of the
  transcript models and a TSV file that assigns reads that supported each
  transcript model.
}
\inclimgsm{overview_src/challenge2-flow.pdf}{Supplementary Fig. 5.}{
  Flow diagram of Challenge 2: Transcript isoform quantification. Samples,
  library prep methods, and sequencing platforms used in the challenge are
  indicated at the top. Participants select which data category, library prep,
  and sequencing platform to analyze, run their pipelines to generate
  transcript predictions, and submit an entry which includes predictions for
  all samples. The entries include a GTF file of the transcript models that
  are quantified and a TSV file of the expression quantification. The H1 and
  endodermal cell samples were released after the initial submission deadline
  and participants were required to submit the quantification after the
  deadline.
}
\inclimgsm{overview_src/challenge3-flow.pdf}{Supplementary Fig. 6.}{
  Flow diagram of Challenge 3. Samples, library prep methods, and sequencing
  platforms used in the challenge are indicated at the top. Participants
  select which data category and sequencing platform to analyze, run their
  pipelines to generate transcript predictions, and submit an entry which
  includes predictions for all samples. The entries include a FASTA file of
  the transcript models and a TSV file that assigns reads that supported each
  transcript model.
}
\inclimg{overview_src/challenge1-eval.pdf}{Supplementary Fig. 7.}{
  Flow diagram of the evaluation for Challenge 1. Benchmarks and additional
  orthogonal data that was used for the evaluation are indicated. For example,
  CAGE and QuantSeq data from WTC11 cells were generated and made available
  only after participant submissions; therefore, they represent “hidden”
  data. These was used to define 5’ transcript starts and 3’ ends.
}
\inclimg{overview_src/challenge2-eval.pdf}{Supplementary Fig. 8.}{
  Flow diagram of the evaluation for Challenge 2. (A) Evaluation of Challenge
  2 can be separated into metrics when a ground truth is known or a ground
  truth is unknown. (B) Example analyses to evaluate transcript expression
  using the cell mixing experiment. A sample, H1_mix, was initially provided
  for quantification which was a mix of H1 cells and endodermal cells at an
  undisclosed ratio. After the initial submission, the individual H1 and
  endodermal cell samples were released and participants submitted
  quantifications for each.
}
\inclimg{overview_src/challenge3-eval.pdf}{Supplementary Fig. 9.}{
  Flow diagram of the evaluation for Challenge 3. Only SIRVs are available for
  ground truth information. The evaluation was based on a comparative
  assessment of the predictions followed by targeting specific candidates for
  further validation.
}
%%%%
%% validation
%%%%
\inclimgsz{validation_src/validation-flow.pdf}{140mm}{Supplementary Fig. 10.}{
  Experimental validation approaches for the LRGASP challenges. (A) Multiple
  categories of types of transcript were selected for validation (shown in
  green boxes). These loci will be viewed in the UCSC Genome Browser along
  with additional datasets to aid in the manual design of primers. Amplicons
  will be analyzed by fragment size and pooled to perform long-read sequencing
  with PacBio and ONT (B) A select number of genes were selected for
  transcript isoform-specific qPCR. A combination of probes detecting
  constitutive and alternative regions will be used. (C) RT-PCR validation
  will be performed similar to Challenge 1, except transcript were selected
  from well-studied mammalian immune-related genes.
}
\inclimg{validation_src/primer-design-example.pdf}{Supplementary Fig. 11.}{
  Designing validation primers. a) An example of a unique intron in transcript
  NNC_381534 to validation.  The green and blue region vertical highlights
  indicate the manually selected primer pair regions.  The 'Targets' track,
  produced by Primers-Juju, recapitulates the region as blue item B2M+1, and
  transcript with the maximal possible amplicon drawn in thick boxes. b) The
  Primers-Juju track hub with the addition of the primer pairs design. This
  adds Primer3 results (Primers track) and the most stable primer along with
  the amplicon sequence for the target transcript (Amplicons track).
}


%%%%
%% challenge 1
%%%%

%%%%
%% challenge 2
%%%%



% Extended Data Fig. 54 in stage2_submission_version
\inclimg{challenge2_src/overall_irreproducibility.pdf}{Supplementary Fig. 1_overall_irreproducibility}{
  Overall evaluation results of irreproducibility on real data with
  multiple replicates.  The diagram illustrates the calculation of
  irreproducibility. By fitting the coefficient of variation (CV)
  versus average transcript abundance into a smooth curve, it can be
  shown that Method X has lower coefficient of variation and higher
  reproducibility.  Evaluation results of ACVC metric for different
  quantification tools and protocols-platforms. Box plots are employed
  to illustrate the five-number summary of evaluation results across
  various datasets, depicting the minimum, lower quartile, median,
  upper quartile, and maximum values.  The overall results of CV
  curves with different transcript abundances on four samples (H1-mix,
  WTC11, H1-hESC and H1-DE) with different protocols and
  platforms. Here, Bambu-merge represents the transcript
  quantification using Bambu with GENCODE plus LR-specific
  annotation. And Bambu-LR represents the transcript quantification
  using only LR-specific annotation.
}
% Extended Data Fig. 55 in stage2_submission_version
\inclimg{challenge2_src/overall_Consistency.pdf}{Supplementary Fig. 2_overall_Consistency.}{
  Overall evaluation results of consistency on real data with multiple
  replicates.  a) The diagram illustrates the calculation of
  consistency. By setting an expression threshold (i.e. 1 in this toy
  example), we can define which set of transcripts express (in blue)
  or not (in orange). This statistic is to measure the consistency of
  the expressed transcripts sets between replicates.  b) A toy example
  to show the consistency curves with different abundance
  threshold. Here, method X performs the better consistency of
  transcript abundance estimation across multiple replicates than
  method Y.  c) Evaluation results of ACC metric for different
  quantification tools and protocols-platforms. Box plots are employed
  to illustrate the five-number summary of evaluation results across
  various datasets, depicting the minimum, lower quartile, median,
  upper quartile, and maximum values.  d) The detailed evaluation
  results of consistency curves with different abundance thresholds on
  four samples (H1-mix, WTC11, H1-hESC and H1-DE) with different
  protocols and platforms.
}
% Extended Data Fig. 56 in stage2_submission_version
\inclimg{challenge2_src/resolution_entropy_description.pdf}{Supplementary Fig. 3_Resolution_Entropy_description.}{
  Resolution Entropy.  a) The software output only a few certain
  discrete values has lower resolution entropy as it cannot capture
  the continuous and subtle difference of gene expressions.  b) The
  software with continuous output values has higher resolution entropy.
}
% Extended Data Fig. 57 in stage2_submission_version
\inclimg{challenge2_src/overall_cell_mixing_experiment.pdf}{Supplementary Fig. 4_overall_cell_mixing_experiment.}{
  Performance evaluation on cell mixing experiment.  a) Schematic
  diagram of evaluation strategy using the cell mixing
  experiment. Here, H1-mix was initially provided for quantification
  which was a mix of H1-hESC cells and H1-DE cells at an undisclosed
  ratio. After the initial submission, the individual H1-hESC and
  H1-DE samples were released and participants submitted
  quantifications for each.  b) Evaluation results of NRMSE metric for
  different quantification tools and protocols-platforms. Bar plots
  are utilized to visualize the mean values of evaluation results
  across diverse datasets, with error bars indicating the standard
  deviation of metrics.  c) Scatter plot of expected abundance and
  observed abundance for seven participant's tools with different
  protocols and platforms.
}
% Extended Data Fig. 58 in stage2_submission_version
\inclimg{challenge2_src/overall_SIRV-set4_data.pdf}{Supplementary Fig. 5_overall_SIRV-set4_data.}{
  Performance evaluation on SIRV-set 4 data.  a) Evaluation results of
  NRMSE metric for different quantification tools and
  protocols-platforms. Bar plots are utilized to visualize the mean
  values of evaluation results across diverse datasets, with error
  bars indicating the standard deviation of NRMSE metric.  b) Scatter
  plot of true abundance and estimated abundance on SIRV-set 4 data
  with different protocols and platforms.
}
% Extended Data Fig. 59 in stage2_submission_version
\inclimg{challenge2_src/overall_simulation_data.pdf}{Supplementary Fig. 6_overall_simulation_data.}{
  Performance evaluation on simulation data.  a) The flow chart of
  simulation study.  b) Evaluation results of NRMSE metric for
  different quantification tools and protocols-platforms. Bar plots
  are utilized to visualize the mean values of evaluation results
  across diverse datasets, with error bars indicating the standard
  deviation of NRMSE metric.  c) Scatter plot of true abundance and
  estimated abundance on simulation data.
}

% Extended Data Fig. 60 in stage2_submission_version
\inclimg{challenge2_src/unaccurate_annotaion_impact_on_quantification.pdf}{Supplementary Fig. 7_unaccurate_annotaion_impact_on_quantification.}{
  Impact of annotation accuracy on transcript quantification. We
  assessed the performance of RSEM and LR-based tools (Bambu, FLAIR,
  FLAMES, IsoQuant, IsoTools, TALON, and NanoSim) with different
  annotations. The NRMSE metric was used to evaluate their performance
  on simulated data for human and mouse. For LR-based tools, the
  transcript quantification annotations were derived from
  sample-specific annotations identified by the participant using
  long-read RNA-seq data. As for RSEM, we present quantification
  results based on two annotations: a completely accurate annotation
  (i.e., the ground truth transcripts generated by the simulation
  data) and an inaccurate annotation (i.e., the common GENCODE
  reference annotation, which contains numerous false negative and
  false positive transcripts specific to the sample). Bar plots are
  utilized to visualize the mean values of evaluation results across
  diverse datasets, with error bars indicating the standard deviation
  of NRMSE metric.
}

% Extended Data Fig. 61 in stage2_submission_version
\inclimg{challenge2_src/length_distribution.pdf}{Supplementary Fig. 8_length_distribution.}{
  Read length distributions in six protocols-platforms.
}
% Extended Data Fig. 62 in stage2_submission_version
\inclimg{challenge2_src/K-value_description.pdf}{Supplementary Fig. 9_K-value_description.}{
  Description of K-value.  A measure of the complexity of exon-isoform
  structures for each gene.
}


%%%%
%% challenge 3
%%%%
\end{document}
